src/modules/commander/PreflightCheck.cpp

```diff
@@ -367,11 +388,14 @@ static bool gyroCheck(orb_advert_t *mavlink_log_pub, unsigned instance, bool opt
 	}
 
 out:
+	if(instance==0) set_health_flags(subsystem_info_s::SUBSYSTEM_TYPE_GYRO, present, !optional, success, status);
```

Comment:
Code style: Should be `if () { }`, not  `if(`

Reply:
Changed. Why is the auto-formatter not catching this?

Reply:
Commander hasn't been included in the auto-formatter(to limit churn), but will be in the future as it's refactored.